lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp

```diff
@@ -1159,6 +1159,14 @@ static std::string GetPluginServerForSDK(llvm::StringRef sdk_path) {
   return server_or_err->str();
 }
 
+namespace {
+  static constexpr std::array<std::string_view, 4> knownExplicitModulePrefixes =
```

Comment:
```suggestion
  static constexpr std::array<std::string_view, 4> g_known_eplicit_module_prefixes =
```
to make this conform to LLDB naming conventions.